src/app/Library/CrudPanel/CrudField.php

```diff
@@ -481,6 +481,11 @@ public function dd()
     public function __call($method, $parameters)
     {
         if (static::hasMacro($method)) {
+            if (in_array($method, app('UploadersRepository')->getUploadersGroupsNames())) {
+                $this->setAttributeValue($method, $parameters[0] ?? true);
+                $this->save();
+            }
+
```

Comment:
Hey @pxpm - could you please explain why this is needed?

It stands out to me because it makes Uploaders behave in a non-standard way. They are ABOVE macros. We have an API for developers to create `$column->something()` and `$field->something()` but we're not using it ourselves. Why? Is it not possible for each Uploader to register its own macro? Is this a lifecycle thing?

Thanks!

Reply:
Thanks for spotting it, indeed this does not need to be here. We already have the macros in place. 👍 

Just refactored this bit of code to the macros here: https://github.com/Laravel-Backpack/CRUD/pull/5478/commits/17f353b27e97675a8d9a71e98d296aed87c58d17

Cheers